eq-author-api/README.md

```diff
@@ -57,48 +65,17 @@ docker-compose build
 docker-compose up
 ```
 
-Once the containers are running you should be able to navigate to http://localhost:4000/graphiql and begin exploring the eQ Author GraphQL API.
-
 Changes to the application should hot reload via `nodemon`.
 
-### Querying pages
-
-There is no concrete `Page` type in the GraphQL schema. Instead we use a `Page` interface, which other types implement e.g. `QuestionPage` and `InterstitialPage`.
-
-To query all pages, and request different fields depending on the type, use [inline fragments](http://graphql.org/learn/queries/#inline-fragments):
-
-```gql
-query {
-  getQuestionnaire(id: 1) {
-    questionnaire {
-      sections {
-        pages {
-          id
-
-          # inline fragment for `QuestionPage` type
-          ... on QuestionPage {
-            guidance
-            answers {
-              id
-              label
-            }
-          }
-
-          # For purposes of example only. `InterstitialPage` doesn't exist yet
-          ... on InterstitialPage {
-            # doesn't exist yet
-            someField
-          }
-        }
-      }
-    }
-  }
-}
-```
+### View survey gives Error 500 (on Mac)
```

Comment:
When were you getting this? I'd rather not have a bug described in here and rather our setup docs ensured that this wasn't possible.

Reply:
Agree with @tgandrews here. The user should not get a 500, so this feels like a bug to me. Feel free to capture in an issue instead?

Reply:
Got this error when I set up the project. Richard advised me how to fix it as he had the same issue.

I will remove it from the docs.

Reply:
This has been resolved.